std_msgs/Header header

ObjectDescription description

string status
string bulb_status
string functioning_status
string behaviour
string traffic_signal_status

float32 azimuth_angle
float32 facing
